platform/mac/mac.gradle

```diff
@@ -54,7 +54,9 @@ tasks.register('exportMacX64', Copy.class) {
     into(exportPathX64 + '/Contents/')
     into('') {
         from file('platform/mac/Info.plist') rename('Info.plist', 'Info.plistdos') filter { line ->
-            line.replace('%mcreator%', (String) project.mcreatorconf.getProperty('mcreator'))
+            line = line.replace('%mcreator%', (String) project.mcreatorconf.getProperty('mcreator'))

```

Comment:
Usually we just chain the replace calls to omit return line:

![image](https://github.com/MCreator/MCreator/assets/16374228/99ded629-47e8-465c-bf61-9a6f0f89821b)

Reply:
Fixed